src/notfound/handler.js

```diff
@@ -49,15 +84,16 @@ export default async function audit404(message, context) {
 
     const data = await rumAPIClient.get404Sources(params);
     const auditResult = process404Response(data);
+    await processAuditResult(dataAccess, site, auditContext, auditResult);
 
     await sqs.sendMessage(queueUrl, {
       type,
-      url,
+      url: siteId,
```

Comment:
I think here you have to pass the `site.getBaseURL()` not the site id, judging by the unit test